src/app/landing/PartnerPicture.jsx

```diff
@@ -1,12 +1,33 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './PartnerPicture.css';
 
-const PartnerPicture = ({ src, alt, link }) => (
+const PartnerPicture = ({ img, alt = '', link }) => (
   <div className="PartnerPicture">
     <a href={link}>
-      <img src={src} alt={alt} />
+      <picture>
+        <source
+          className="OnboardingImage"
+          media="(min-aspect-ratio: 2/3)"
```

Comment:
For the map picture, we used the ratio as the deciding factor because it was the reason we had 2 versions in the first place: centering the markers on the map in both "portrait" and "landscape" displays, while keeping it full-width, was difficult with a single version.

However, since the factor that makes us use 2 versions in this case is actually the resolution/display size, the breakpoint should be based on that as well, and not on the ratio (while it's a good hueristic that phones are more "vertical" than desktops, a phone rotated sideways, for example, still doesn't need a large version of a picture).

(That of course goes for all the new assets, not just this place.)

Reply:
Sounds good, do you or Di have a preference for a specific breakpoint to use for mobile?

If not, I will go ahead and try

/* Smartphones (portrait) ----------- */
@media only screen and (max-width : 320px) {
/* Styles */
}
as set by https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/

Reply:
I'm not actually sure it should be based on height/width, as opposed to resolution ("device-pixel-ratio" or whatnot). Looks like the `picture` element already has `srcset` with the ability to specify different pictures for "1x", "2x" etc, which sounds more correct.

But, I haven't looked into it too much. There's probably posts out there with best practices, so I'd just find a reliable one and follow whatever it says.